internal/handlers/hana/hana.go

```diff
@@ -110,6 +111,16 @@ func (h *Handler) DBPool(ctx context.Context) (*hanadb.Pool, error) {
 	return p, nil
 }
 
+// Describe implements handlers.Interface.
+func (h *Handler) Describe(ch chan<- *prometheus.Desc) {
+	// TODO
```

Comment:
Any issue number for those?

Reply:
No, because we are going to merge Hana and SQLite handlers soon, and other handlers will be migrated to the universal handler too. And SQLite handler will have code there in the next PR